README.md

```diff
@@ -237,55 +284,66 @@ A set of guidelines for a specific programming language that recommend programmi
 - [BEM - Block Element Modifier](https://getbem.com/) - A methodology that helps you to create reusable components and code sharing in front‑end development.
 
 ### GNU
+
 - [GNU coding standards](https://www.gnu.org/prep/standards/)
 - [Making The Best Use of C](https://www.gnu.org/prep/standards/html_node/Writing-C.html) - This chapter provides advice on how best to use the C language when writing GNU software.
 
 ### Java
+
 - [JavaEE Specification](https://github.com/javaee/javaee-spec)
 
 ### Linux
+
 - [Linux kernel coding style](https://www.kernel.org/doc/html/latest/process/coding-style.html)
 
 ### Mailchimp
+
 - [Mailchimp Content Style Guide](https://styleguide.mailchimp.com)
 
 ### Mozilla
+
 - [Mozilla Coding Style Guide](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide)
 
 ### Google
+
 - [Google-related developer documentation](https://developers.google.com/style) - This style guide provides editorial guidelines for writing clear and consistent Google-related developer documentation.
 
 ### Node.js
+
 - [npm-coding-style](https://docs.npmjs.com/misc/coding-style)
 - [Microsoft + Node.js Guidelines](https://github.com/Microsoft/nodejs-guidelines)
 - [Node.js Style Guide](https://github.com/felixge/node-style-guide) - A guide for styling your Node.js / JavaScript code.
 
 ### MongoDB
+
 - [Mongo Style Guide](https://github.com/jsoendermann/MongoStyleGuide)
 
 ### SQL
+
 - [SQL Style Guide](https://www.sqlstyle.guide)
 
 ### Other
+
 - [Keep a CHANGELOG](http://keepachangelog.com/en/0.3.0/) - Don't let your friends dump git logs into CHANGELOGs™.
 - [Project Guidelines](https://github.com/wearehive/project-guidelines) - A set of best practices for JavaScript projects.
 - [Semantic Versioning](http://semver.org)
 - [Indent style](https://en.wikipedia.org/wiki/Indent_style)
 - [WebAppSec/Secure Coding Guidelines](https://wiki.mozilla.org/WebAppSec/Secure_Coding_Guidelines)
 - [Robot Framework User Guide](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html)
 
-
 ## Frameworks
+
 - [Symfony Coding Standards](https://symfony.com/doc/current/contributing/code/standards.html)
 - [Django Coding Style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style)
-
+- [Vue Style Guide](https://vuejs.org/guide/introduction.html)
```

Comment:
I don't see this as coding style conventions or standards, just documentation.

If you want, you can change the link: https://vuejs.org/style-guide